Issue when processing RNS Events

When registering a new Domain running locally against `ganache`, we get the following error.

```
[ERROR] 04/01 03:33:39 (web3events:rns:): Error in the processing loop:
{}
[ERROR] 04/01 03:33:39 (rns): There was unknown error in Events Emitter for Error! undefined
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
```

The following is the log in `DEBUG` mode. Contract Addresses are OK, using the `ganache.json5` config file.

```

[INFO] 04/01 03:33:34 (web3events:rns.owner:): Fetching and processing events from block 1995 to 1995
[DEBUG] 04/01 03:33:34 (web3events:rns.owner:): Batch size is 120
[VERBOSE] 04/01 03:33:34 (web3events:rns.owner:confirmator): Running Confirmation routine
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
SELECT
  `id`,
  `blockNumber`,
  `transactionHash`,
  `targetConfirmation`,
  `contractAddress`,
  `event`,
  `content`,
  `emitted`,
  `createdAt`,
  `updatedAt`
FROM
  `web3events_event` AS `web3events_event`
WHERE
  `web3events_event`.`contractAddress` = '0x26b4afb60d6c903165150c6f0aa14f8016be4aec'
  AND `web3events_event`.`emitted` = 0
  AND `web3events_event`.`blockNumber` <= 1995;
[INFO] 04/01 03:33:34 (web3events:storage.storageManager:confirmator): Confirmed 0 events.
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (NULL)
[VERBOSE] 04/01 03:33:34 (web3events:storage.storageManager:): Will process 1 batches
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
DELETE FROM
  `web3events_event`
WHERE
  `emitted` = 1
  AND `contractAddress` = '0x4339316e04cffb5961d1c41fef8e44bfa2a7fbd1'
  AND `blockNumber` <= 1985 - `targetConfirmation`
[INFO] 04/01 03:33:34 (web3events:storage.staking:confirmator): Confirmed 0 events.
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (NULL)
[VERBOSE] 04/01 03:33:34 (web3events:storage.staking:): Will process 1 batches
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
DELETE FROM
  `web3events_event`
WHERE
  `emitted` = 1
  AND `contractAddress` = '0x26b4afb60d6c903165150c6f0aa14f8016be4aec'
  AND `blockNumber` <= 1985 - `targetConfirmation`
[DEBUG] 04/01 03:33:34 (web3events:storage.storageManager:): Received events for the batch: 
[INFO] 04/01 03:33:34 (web3events:storage.storageManager:): No events to be processed.
[INFO] 04/01 03:33:34 (web3events:storage.storageManager:): Finished fetching events in 0s
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.storageManager.lastFetchedBlockHash',
    '0x52ea50cc996bce2e8bb26623a5ddbd22dd44ea8c69f50c1e60d70334acde8cc8'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[INFO] 04/01 03:33:34 (web3events:rns.owner:confirmator): Confirmed 0 events.
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (NULL)
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.storageManager.lastFetchedBlockNumber',
    '1995'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[VERBOSE] 04/01 03:33:34 (web3events:rns.owner:): Will process 1 batches
[DEBUG] 04/01 03:33:34 (web3events:storage.staking:): Received events for the batch: 
[INFO] 04/01 03:33:34 (web3events:storage.staking:): No events to be processed.
[INFO] 04/01 03:33:34 (web3events:storage.staking:): Finished fetching events in 0s
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.staking.lastFetchedBlockHash',
    '0x52ea50cc996bce2e8bb26623a5ddbd22dd44ea8c69f50c1e60d70334acde8cc8'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.staking.lastFetchedBlockNumber',
    '1995'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:34 (web3events:rns.owner:): Received events for the batch: 
[INFO] 04/01 03:33:34 (web3events:rns.owner:): No events to be processed.
[VERBOSE] 04/01 03:33:34 (web3events:rns.reverse:): Lock acquired for fetch()
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.rns.owner.lastFetchedBlockHash',
    '0x52ea50cc996bce2e8bb26623a5ddbd22dd44ea8c69f50c1e60d70334acde8cc8'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.rns.owner.lastFetchedBlockNumber',
    '1995'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[INFO] 04/01 03:33:34 (web3events:rns.reverse:): Fetching and processing events from block 1995 to 1995
[DEBUG] 04/01 03:33:34 (web3events:rns.reverse:): Batch size is 120
[VERBOSE] 04/01 03:33:34 (web3events:rns.reverse:confirmator): Running Confirmation routine
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
SELECT
  `id`,
  `blockNumber`,
  `transactionHash`,
  `targetConfirmation`,
  `contractAddress`,
  `event`,
  `content`,
  `emitted`,
  `createdAt`,
  `updatedAt`
FROM
  `web3events_event` AS `web3events_event`
WHERE
  `web3events_event`.`contractAddress` = '0x9561c133dd8580860b6b7e504bc5aa500f0f06a7'
  AND `web3events_event`.`emitted` = 0
  AND `web3events_event`.`blockNumber` <= 1995;
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
DELETE FROM
  `web3events_event`
WHERE
  `emitted` = 1
  AND `contractAddress` = '0x9561c133dd8580860b6b7e504bc5aa500f0f06a7'
  AND `blockNumber` <= 1985 - `targetConfirmation`
[INFO] 04/01 03:33:34 (web3events:rns.reverse:confirmator): Confirmed 0 events.
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (NULL)
[VERBOSE] 04/01 03:33:34 (web3events:rns.reverse:): Will process 1 batches
[DEBUG] 04/01 03:33:34 (web3events:rns.reverse:): Received events for the batch: 
[INFO] 04/01 03:33:34 (web3events:rns.reverse:): No events to be processed.
[VERBOSE] 04/01 03:33:34 (web3events:rns.placement:): Lock acquired for fetch()
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.rns.reverse.lastFetchedBlockHash',
    '0x52ea50cc996bce2e8bb26623a5ddbd22dd44ea8c69f50c1e60d70334acde8cc8'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.rns.reverse.lastFetchedBlockNumber',
    '1995'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[INFO] 04/01 03:33:34 (web3events:rns.placement:): Fetching and processing events from block 1995 to 1995
[DEBUG] 04/01 03:33:34 (web3events:rns.placement:): Batch size is 120
[VERBOSE] 04/01 03:33:34 (web3events:rns.placement:confirmator): Running Confirmation routine
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
SELECT
  `id`,
  `blockNumber`,
  `transactionHash`,
  `targetConfirmation`,
  `contractAddress`,
  `event`,
  `content`,
  `emitted`,
  `createdAt`,
  `updatedAt`
FROM
  `web3events_event` AS `web3events_event`
WHERE
  `web3events_event`.`contractAddress` = '0xef391648603dcb012e30ef706452b3dff6466de8'
  AND `web3events_event`.`emitted` = 0
  AND `web3events_event`.`blockNumber` <= 1995;
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
DELETE FROM
  `web3events_event`
WHERE
  `emitted` = 1
  AND `contractAddress` = '0xef391648603dcb012e30ef706452b3dff6466de8'
  AND `blockNumber` <= 1985 - `targetConfirmation`
[INFO] 04/01 03:33:34 (web3events:rns.placement:confirmator): Confirmed 0 events.
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (NULL)
[VERBOSE] 04/01 03:33:34 (web3events:rns.placement:): Will process 1 batches
[DEBUG] 04/01 03:33:34 (web3events:rns.placement:): Received events for the batch: 
[INFO] 04/01 03:33:34 (web3events:rns.placement:): No events to be processed.
[INFO] 04/01 03:33:34 (web3events:rns.owner:): Finished fetching events in 0s
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.rns.placement.lastFetchedBlockHash',
    '0x52ea50cc996bce2e8bb26623a5ddbd22dd44ea8c69f50c1e60d70334acde8cc8'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[INFO] 04/01 03:33:34 (web3events:rns.reverse:): Finished fetching events in 0s
[INFO] 04/01 03:33:34 (web3events:rns.placement:): Finished fetching events in 0s
[DEBUG] 04/01 03:33:34 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.rns.placement.lastFetchedBlockNumber',
    '1995'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:39 (blockchain): New block: 
{
  number: 1996,
  hash: '0xe8e1275df49da4cc90c4a7878feb13b1870f22f432c1d485053cdc6d6dfbd5dc',
  parentHash: '0x52ea50cc996bce2e8bb26623a5ddbd22dd44ea8c69f50c1e60d70334acde8cc8',
  mixHash: '0x0000000000000000000000000000000000000000000000000000000000000000',
  nonce: '0x0000000000000000',
  sha3Uncles: '0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347',
  logsBloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
  transactionsRoot: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
  stateRoot: '0x1eaf5ec4a021382eafa871461bcb0553c146b06c1d4b7e4e8cbe9862625887ac',
  receiptsRoot: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
  miner: '0x0000000000000000000000000000000000000000',
  difficulty: '0',
  totalDifficulty: '0',
  extraData: '0x',
  size: 1000,
  gasLimit: 6721975,
  gasUsed: 0,
  timestamp: 1609785218,
  transactions: [],
  uncles: []
}
[VERBOSE] 04/01 03:33:39 (web3events:rns:): Received new block number 1996
[VERBOSE] 04/01 03:33:39 (web3events:storage.storageManager:): Received new block number 1996
[VERBOSE] 04/01 03:33:39 (web3events:storage.staking:): Received new block number 1996
[VERBOSE] 04/01 03:33:39 (web3events:storage.storageManager:): Lock acquired for fetch()
[VERBOSE] 04/01 03:33:39 (web3events:storage.staking:): Lock acquired for fetch()
[VERBOSE] 04/01 03:33:39 (web3events:rns.owner:): Lock acquired for fetch()
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'blockchain.lastFetchedBlock',
    '{"number":1996,"hash":"0xe8e1275df49da4cc90c4a7878feb13b1870f22f432c1d485053cdc6d6dfbd5dc","parentHash":"0x52ea50cc996bce2e8bb26623a5ddbd22dd44ea8c69f50c1e60d70334acde8cc8","mixHash":"0x0000000000000000000000000000000000000000000000000000000000000000","nonce":"0x0000000000000000","sha3Uncles":"0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347","logsBloom":"0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","transactionsRoot":"0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421","stateRoot":"0x1eaf5ec4a021382eafa871461bcb0553c146b06c1d4b7e4e8cbe9862625887ac","receiptsRoot":"0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421","miner":"0x0000000000000000000000000000000000000000","difficulty":"0","totalDifficulty":"0","extraData":"0x","size":1000,"gasLimit":6721975,"gasUsed":0,"timestamp":1609785218,"transactions":[],"uncles":[]}'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[INFO] 04/01 03:33:39 (web3events:storage.storageManager:): Fetching and processing events from block 1996 to 1996
[DEBUG] 04/01 03:33:39 (web3events:storage.storageManager:): Batch size is 120
[VERBOSE] 04/01 03:33:39 (web3events:storage.storageManager:confirmator): Running Confirmation routine
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
SELECT
  `id`,
  `blockNumber`,
  `transactionHash`,
  `targetConfirmation`,
  `contractAddress`,
  `event`,
  `content`,
  `emitted`,
  `createdAt`,
  `updatedAt`
FROM
  `web3events_event` AS `web3events_event`
WHERE
  `web3events_event`.`contractAddress` = '0xddb64fe46a91d46ee29420539fc25fd07c5fea3e'
  AND `web3events_event`.`emitted` = 0
  AND `web3events_event`.`blockNumber` <= 1996;
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
DELETE FROM
  `web3events_event`
WHERE
  `emitted` = 1
  AND `contractAddress` = '0xddb64fe46a91d46ee29420539fc25fd07c5fea3e'
  AND `blockNumber` <= 1986 - `targetConfirmation`
[INFO] 04/01 03:33:39 (web3events:storage.staking:): Fetching and processing events from block 1996 to 1996
[DEBUG] 04/01 03:33:39 (web3events:storage.staking:): Batch size is 120
[VERBOSE] 04/01 03:33:39 (web3events:storage.staking:confirmator): Running Confirmation routine
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
SELECT
  `id`,
  `blockNumber`,
  `transactionHash`,
  `targetConfirmation`,
  `contractAddress`,
  `event`,
  `content`,
  `emitted`,
  `createdAt`,
  `updatedAt`
FROM
  `web3events_event` AS `web3events_event`
WHERE
  `web3events_event`.`contractAddress` = '0x4339316e04cffb5961d1c41fef8e44bfa2a7fbd1'
  AND `web3events_event`.`emitted` = 0
  AND `web3events_event`.`blockNumber` <= 1996;
[INFO] 04/01 03:33:39 (web3events:rns.owner:): Fetching and processing events from block 1996 to 1996
[DEBUG] 04/01 03:33:39 (web3events:rns.owner:): Batch size is 120
[VERBOSE] 04/01 03:33:39 (web3events:rns.owner:confirmator): Running Confirmation routine
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
SELECT
  `id`,
  `blockNumber`,
  `transactionHash`,
  `targetConfirmation`,
  `contractAddress`,
  `event`,
  `content`,
  `emitted`,
  `createdAt`,
  `updatedAt`
FROM
  `web3events_event` AS `web3events_event`
WHERE
  `web3events_event`.`contractAddress` = '0x26b4afb60d6c903165150c6f0aa14f8016be4aec'
  AND `web3events_event`.`emitted` = 0
  AND `web3events_event`.`blockNumber` <= 1996;
[INFO] 04/01 03:33:39 (web3events:storage.storageManager:confirmator): Confirmed 0 events.
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (NULL)
[VERBOSE] 04/01 03:33:39 (web3events:storage.storageManager:): Will process 1 batches
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
DELETE FROM
  `web3events_event`
WHERE
  `emitted` = 1
  AND `contractAddress` = '0x4339316e04cffb5961d1c41fef8e44bfa2a7fbd1'
  AND `blockNumber` <= 1986 - `targetConfirmation`
[INFO] 04/01 03:33:39 (web3events:storage.staking:confirmator): Confirmed 0 events.
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (NULL)
[VERBOSE] 04/01 03:33:39 (web3events:storage.staking:): Will process 1 batches
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
DELETE FROM
  `web3events_event`
WHERE
  `emitted` = 1
  AND `contractAddress` = '0x26b4afb60d6c903165150c6f0aa14f8016be4aec'
  AND `blockNumber` <= 1986 - `targetConfirmation`
[INFO] 04/01 03:33:39 (web3events:rns.owner:confirmator): Confirmed 3 events.
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (7, 8, 9)
[DEBUG] 04/01 03:33:39 (web3events:storage.storageManager:): Received events for the batch: 
[INFO] 04/01 03:33:39 (web3events:storage.storageManager:): No events to be processed.
[INFO] 04/01 03:33:39 (web3events:storage.storageManager:): Finished fetching events in 0s
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.storageManager.lastFetchedBlockHash',
    '0xe8e1275df49da4cc90c4a7878feb13b1870f22f432c1d485053cdc6d6dfbd5dc'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:39 (web3events:rns.owner:confirmator): Confirming event
{
  logIndex: 2,
  transactionIndex: 0,
  transactionHash: '0x7a2da2aa28ec3d4ad52f2b5c99d5c016caed34f174b686075a160304216d134d',
  blockHash: '0xaa5035291ce0dc8c333d1c55f557996d778382bcc388d692a42aecb933b60796',
  blockNumber: 1994,
  address: '0x26b4AFb60d6C903165150C6F0AA14F8016bE4aec',
  type: 'mined',
  id: 'log_32c88a00',
  returnValues: {
    '0': '51417028818506933953789413346456257902654491136419453012810895738703409414510',
    '1': '1704393201',
    tokenId: '51417028818506933953789413346456257902654491136419453012810895738703409414510',
    expirationTime: '1704393201'
  },
  event: 'ExpirationChanged',
  signature: '0x9aa6fad74a092697838e399f30ea908478bb2747b84bedf64798609770f44587',
  raw: {
    data: '0x71ad01cb6fd105d46d2859e07f154c2116f2b7c1433e60dfe71b11fe8b31816e000000000000000000000000000000000000000000000000000000006596f9f1',
    topics: [
      '0x9aa6fad74a092697838e399f30ea908478bb2747b84bedf64798609770f44587'
    ]
  }
}
[DEBUG] 04/01 03:33:39 (web3events:rns.owner:confirmator): Confirming event
{
  logIndex: 3,
  transactionIndex: 0,
  transactionHash: '0x7a2da2aa28ec3d4ad52f2b5c99d5c016caed34f174b686075a160304216d134d',
  blockHash: '0xaa5035291ce0dc8c333d1c55f557996d778382bcc388d692a42aecb933b60796',
  blockNumber: 1994,
  address: '0x26b4AFb60d6C903165150C6F0AA14F8016bE4aec',
  type: 'mined',
  id: 'log_68114495',
  returnValues: {
    '0': '0x0000000000000000000000000000000000000000',
    '1': '0x6eD79Aa1c71FD7BdBC515EfdA3Bd4e26394435cC',
    '2': '51417028818506933953789413346456257902654491136419453012810895738703409414510',
    from: '0x0000000000000000000000000000000000000000',
    to: '0x6eD79Aa1c71FD7BdBC515EfdA3Bd4e26394435cC',
    tokenId: '51417028818506933953789413346456257902654491136419453012810895738703409414510'
  },
  event: 'Transfer',
  signature: '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef',
  raw: {
    data: '0x',
    topics: [
      '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef',
      '0x0000000000000000000000000000000000000000000000000000000000000000',
      '0x0000000000000000000000006ed79aa1c71fd7bdbc515efda3bd4e26394435cc',
      '0x71ad01cb6fd105d46d2859e07f154c2116f2b7c1433e60dfe71b11fe8b31816e'
    ]
  }
}
[DEBUG] 04/01 03:33:39 (web3events:rns.owner:confirmator): Confirming event
{
  logIndex: 8,
  transactionIndex: 0,
  transactionHash: '0x7a2da2aa28ec3d4ad52f2b5c99d5c016caed34f174b686075a160304216d134d',
  blockHash: '0xaa5035291ce0dc8c333d1c55f557996d778382bcc388d692a42aecb933b60796',
  blockNumber: 1994,
  address: '0x26b4AFb60d6C903165150C6F0AA14F8016bE4aec',
  type: 'mined',
  id: 'log_8288a1f7',
  returnValues: {
    '0': '0x6eD79Aa1c71FD7BdBC515EfdA3Bd4e26394435cC',
    '1': '0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1',
    '2': '51417028818506933953789413346456257902654491136419453012810895738703409414510',
    from: '0x6eD79Aa1c71FD7BdBC515EfdA3Bd4e26394435cC',
    to: '0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1',
    tokenId: '51417028818506933953789413346456257902654491136419453012810895738703409414510'
  },
  event: 'Transfer',
  signature: '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef',
  raw: {
    data: '0x',
    topics: [
      '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef',
      '0x0000000000000000000000006ed79aa1c71fd7bdbc515efda3bd4e26394435cc',
      '0x00000000000000000000000090f8bf6a479f320ead074411a4b0e7944ea8c9c1',
      '0x71ad01cb6fd105d46d2859e07f154c2116f2b7c1433e60dfe71b11fe8b31816e'
    ]
  }
}
[VERBOSE] 04/01 03:33:39 (web3events:rns.owner:): Emitting Batch with Confirmed events
[VERBOSE] 04/01 03:33:39 (web3events:rns.reverse:): Lock acquired for fetch()
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.storageManager.lastFetchedBlockNumber',
    '1996'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:39 (web3events:storage.staking:): Received events for the batch: 
[INFO] 04/01 03:33:39 (web3events:storage.staking:): No events to be processed.
[INFO] 04/01 03:33:39 (web3events:storage.staking:): Finished fetching events in 0s
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.staking.lastFetchedBlockHash',
    '0xe8e1275df49da4cc90c4a7878feb13b1870f22f432c1d485053cdc6d6dfbd5dc'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.staking.lastFetchedBlockNumber',
    '1996'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[INFO] 04/01 03:33:39 (web3events:rns.reverse:): Fetching and processing events from block 1996 to 1996
[DEBUG] 04/01 03:33:39 (web3events:rns.reverse:): Batch size is 120
[VERBOSE] 04/01 03:33:39 (web3events:rns.reverse:confirmator): Running Confirmation routine
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
SELECT
  `id`,
  `blockNumber`,
  `transactionHash`,
  `targetConfirmation`,
  `contractAddress`,
  `event`,
  `content`,
  `emitted`,
  `createdAt`,
  `updatedAt`
FROM
  `web3events_event` AS `web3events_event`
WHERE
  `web3events_event`.`contractAddress` = '0x9561c133dd8580860b6b7e504bc5aa500f0f06a7'
  AND `web3events_event`.`emitted` = 0
  AND `web3events_event`.`blockNumber` <= 1996;
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
DELETE FROM
  `web3events_event`
WHERE
  `emitted` = 1
  AND `contractAddress` = '0x9561c133dd8580860b6b7e504bc5aa500f0f06a7'
  AND `blockNumber` <= 1986 - `targetConfirmation`
[INFO] 04/01 03:33:39 (web3events:rns.reverse:confirmator): Confirmed 0 events.
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (NULL)
[VERBOSE] 04/01 03:33:39 (web3events:rns.reverse:): Will process 1 batches
[DEBUG] 04/01 03:33:39 (web3events:rns.reverse:): Received events for the batch: 
[INFO] 04/01 03:33:39 (web3events:rns.reverse:): No events to be processed.
[VERBOSE] 04/01 03:33:39 (web3events:rns.placement:): Lock acquired for fetch()
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.rns.reverse.lastFetchedBlockHash',
    '0xe8e1275df49da4cc90c4a7878feb13b1870f22f432c1d485053cdc6d6dfbd5dc'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.rns.reverse.lastFetchedBlockNumber',
    '1996'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[INFO] 04/01 03:33:39 (web3events:rns.placement:): Fetching and processing events from block 1996 to 1996
[DEBUG] 04/01 03:33:39 (web3events:rns.placement:): Batch size is 120
[VERBOSE] 04/01 03:33:39 (web3events:rns.placement:confirmator): Running Confirmation routine
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
SELECT
  `id`,
  `blockNumber`,
  `transactionHash`,
  `targetConfirmation`,
  `contractAddress`,
  `event`,
  `content`,
  `emitted`,
  `createdAt`,
  `updatedAt`
FROM
  `web3events_event` AS `web3events_event`
WHERE
  `web3events_event`.`contractAddress` = '0xef391648603dcb012e30ef706452b3dff6466de8'
  AND `web3events_event`.`emitted` = 0
  AND `web3events_event`.`blockNumber` <= 1996;
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
DELETE FROM
  `web3events_event`
WHERE
  `emitted` = 1
  AND `contractAddress` = '0xef391648603dcb012e30ef706452b3dff6466de8'
  AND `blockNumber` <= 1986 - `targetConfirmation`
[INFO] 04/01 03:33:39 (web3events:rns.placement:confirmator): Confirmed 0 events.
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (NULL)
[VERBOSE] 04/01 03:33:39 (web3events:rns.placement:): Will process 1 batches
[DEBUG] 04/01 03:33:39 (web3events:rns.placement:): Received events for the batch: 
[INFO] 04/01 03:33:39 (web3events:rns.placement:): No events to be processed.
[ERROR] 04/01 03:33:39 (web3events:rns:): Error in the processing loop:
{}
[ERROR] 04/01 03:33:39 (rns): There was unknown error in Events Emitter for Error! undefined
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.rns.placement.lastFetchedBlockHash',
    '0xe8e1275df49da4cc90c4a7878feb13b1870f22f432c1d485053cdc6d6dfbd5dc'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:39 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.rns.placement.lastFetchedBlockNumber',
    '1996'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:49 (blockchain): New block: 
{
  number: 1997,
  hash: '0x867e719c5527980d2ef7902109453112cfc7368684dc82e7dd4f2d6bfeb2559d',
  parentHash: '0xe8e1275df49da4cc90c4a7878feb13b1870f22f432c1d485053cdc6d6dfbd5dc',
  mixHash: '0x0000000000000000000000000000000000000000000000000000000000000000',
  nonce: '0x0000000000000000',
  sha3Uncles: '0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347',
  logsBloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
  transactionsRoot: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
  stateRoot: '0x676ac31d2943779f30f0d1f37f2399a32dcb465626dbfa0d99589671a15fe4d2',
  receiptsRoot: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
  miner: '0x0000000000000000000000000000000000000000',
  difficulty: '0',
  totalDifficulty: '0',
  extraData: '0x',
  size: 1000,
  gasLimit: 6721975,
  gasUsed: 0,
  timestamp: 1609785226,
  transactions: [],
  uncles: []
}
[VERBOSE] 04/01 03:33:49 (web3events:rns:): Received new block number 1997
[VERBOSE] 04/01 03:33:49 (web3events:storage.storageManager:): Received new block number 1997
[VERBOSE] 04/01 03:33:49 (web3events:storage.staking:): Received new block number 1997
[VERBOSE] 04/01 03:33:49 (web3events:storage.storageManager:): Lock acquired for fetch()
[VERBOSE] 04/01 03:33:49 (web3events:storage.staking:): Lock acquired for fetch()
[DEBUG] 04/01 03:33:49 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'blockchain.lastFetchedBlock',
    '{"number":1997,"hash":"0x867e719c5527980d2ef7902109453112cfc7368684dc82e7dd4f2d6bfeb2559d","parentHash":"0xe8e1275df49da4cc90c4a7878feb13b1870f22f432c1d485053cdc6d6dfbd5dc","mixHash":"0x0000000000000000000000000000000000000000000000000000000000000000","nonce":"0x0000000000000000","sha3Uncles":"0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347","logsBloom":"0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","transactionsRoot":"0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421","stateRoot":"0x676ac31d2943779f30f0d1f37f2399a32dcb465626dbfa0d99589671a15fe4d2","receiptsRoot":"0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421","miner":"0x0000000000000000000000000000000000000000","difficulty":"0","totalDifficulty":"0","extraData":"0x","size":1000,"gasLimit":6721975,"gasUsed":0,"timestamp":1609785226,"transactions":[],"uncles":[]}'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[INFO] 04/01 03:33:49 (web3events:storage.storageManager:): Fetching and processing events from block 1997 to 1997
[DEBUG] 04/01 03:33:49 (web3events:storage.storageManager:): Batch size is 120
[VERBOSE] 04/01 03:33:49 (web3events:storage.storageManager:confirmator): Running Confirmation routine
[DEBUG] 04/01 03:33:49 (db): Executing SQL (default):
SELECT
  `id`,
  `blockNumber`,
  `transactionHash`,
  `targetConfirmation`,
  `contractAddress`,
  `event`,
  `content`,
  `emitted`,
  `createdAt`,
  `updatedAt`
FROM
  `web3events_event` AS `web3events_event`
WHERE
  `web3events_event`.`contractAddress` = '0xddb64fe46a91d46ee29420539fc25fd07c5fea3e'
  AND `web3events_event`.`emitted` = 0
  AND `web3events_event`.`blockNumber` <= 1997;
[INFO] 04/01 03:33:49 (web3events:storage.staking:): Fetching and processing events from block 1997 to 1997
[DEBUG] 04/01 03:33:49 (web3events:storage.staking:): Batch size is 120
[VERBOSE] 04/01 03:33:49 (web3events:storage.staking:confirmator): Running Confirmation routine
[DEBUG] 04/01 03:33:49 (db): Executing SQL (default):
SELECT
  `id`,
  `blockNumber`,
  `transactionHash`,
  `targetConfirmation`,
  `contractAddress`,
  `event`,
  `content`,
  `emitted`,
  `createdAt`,
  `updatedAt`
FROM
  `web3events_event` AS `web3events_event`
WHERE
  `web3events_event`.`contractAddress` = '0x4339316e04cffb5961d1c41fef8e44bfa2a7fbd1'
  AND `web3events_event`.`emitted` = 0
  AND `web3events_event`.`blockNumber` <= 1997;
[DEBUG] 04/01 03:33:49 (db): Executing SQL (default):
DELETE FROM
  `web3events_event`
WHERE
  `emitted` = 1
  AND `contractAddress` = '0xddb64fe46a91d46ee29420539fc25fd07c5fea3e'
  AND `blockNumber` <= 1987 - `targetConfirmation`
[INFO] 04/01 03:33:49 (web3events:storage.storageManager:confirmator): Confirmed 0 events.
[DEBUG] 04/01 03:33:49 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (NULL)
[DEBUG] 04/01 03:33:49 (db): Executing SQL (default):
DELETE FROM
  `web3events_event`
WHERE
  `emitted` = 1
  AND `contractAddress` = '0x4339316e04cffb5961d1c41fef8e44bfa2a7fbd1'
  AND `blockNumber` <= 1987 - `targetConfirmation`
[VERBOSE] 04/01 03:33:49 (web3events:storage.storageManager:): Will process 1 batches
[INFO] 04/01 03:33:49 (web3events:storage.staking:confirmator): Confirmed 0 events.
[DEBUG] 04/01 03:33:49 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (NULL)
[VERBOSE] 04/01 03:33:49 (web3events:storage.staking:): Will process 1 batches
[DEBUG] 04/01 03:33:49 (web3events:storage.storageManager:): Received events for the batch: 
[INFO] 04/01 03:33:49 (web3events:storage.storageManager:): No events to be processed.
[INFO] 04/01 03:33:49 (web3events:storage.storageManager:): Finished fetching events in 0s
[DEBUG] 04/01 03:33:49 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.storageManager.lastFetchedBlockHash',
    '0x867e719c5527980d2ef7902109453112cfc7368684dc82e7dd4f2d6bfeb2559d'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:49 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.storageManager.lastFetchedBlockNumber',
    '1997'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:49 (web3events:storage.staking:): Received events for the batch: 
[INFO] 04/01 03:33:49 (web3events:storage.staking:): No events to be processed.
[INFO] 04/01 03:33:49 (web3events:storage.staking:): Finished fetching events in 0s
[DEBUG] 04/01 03:33:49 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.staking.lastFetchedBlockHash',
    '0x867e719c5527980d2ef7902109453112cfc7368684dc82e7dd4f2d6bfeb2559d'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:49 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.staking.lastFetchedBlockNumber',
    '1997'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:54 (blockchain): New block: 
{
  number: 1998,
  hash: '0x17601d78d1afd8f11c0fe638e9cc9180d55dc5d4f5598762a57004a2942b16b1',
  parentHash: '0x867e719c5527980d2ef7902109453112cfc7368684dc82e7dd4f2d6bfeb2559d',
  mixHash: '0x0000000000000000000000000000000000000000000000000000000000000000',
  nonce: '0x0000000000000000',
  sha3Uncles: '0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347',
  logsBloom: '0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
  transactionsRoot: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
  stateRoot: '0x9aeeb9b677d57c2cf626c16f29df542fc15b54e652530dcb2fecf9febc11d7e2',
  receiptsRoot: '0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421',
  miner: '0x0000000000000000000000000000000000000000',
  difficulty: '0',
  totalDifficulty: '0',
  extraData: '0x',
  size: 1000,
  gasLimit: 6721975,
  gasUsed: 0,
  timestamp: 1609785234,
  transactions: [],
  uncles: []
}
[VERBOSE] 04/01 03:33:54 (web3events:rns:): Received new block number 1998
[VERBOSE] 04/01 03:33:54 (web3events:storage.storageManager:): Received new block number 1998
[VERBOSE] 04/01 03:33:54 (web3events:storage.staking:): Received new block number 1998
[VERBOSE] 04/01 03:33:54 (web3events:storage.storageManager:): Lock acquired for fetch()
[VERBOSE] 04/01 03:33:54 (web3events:storage.staking:): Lock acquired for fetch()
[DEBUG] 04/01 03:33:54 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'blockchain.lastFetchedBlock',
    '{"number":1998,"hash":"0x17601d78d1afd8f11c0fe638e9cc9180d55dc5d4f5598762a57004a2942b16b1","parentHash":"0x867e719c5527980d2ef7902109453112cfc7368684dc82e7dd4f2d6bfeb2559d","mixHash":"0x0000000000000000000000000000000000000000000000000000000000000000","nonce":"0x0000000000000000","sha3Uncles":"0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347","logsBloom":"0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000","transactionsRoot":"0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421","stateRoot":"0x9aeeb9b677d57c2cf626c16f29df542fc15b54e652530dcb2fecf9febc11d7e2","receiptsRoot":"0x56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421","miner":"0x0000000000000000000000000000000000000000","difficulty":"0","totalDifficulty":"0","extraData":"0x","size":1000,"gasLimit":6721975,"gasUsed":0,"timestamp":1609785234,"transactions":[],"uncles":[]}'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[INFO] 04/01 03:33:54 (web3events:storage.storageManager:): Fetching and processing events from block 1998 to 1998
[DEBUG] 04/01 03:33:54 (web3events:storage.storageManager:): Batch size is 120
[VERBOSE] 04/01 03:33:54 (web3events:storage.storageManager:confirmator): Running Confirmation routine
[DEBUG] 04/01 03:33:54 (db): Executing SQL (default):
SELECT
  `id`,
  `blockNumber`,
  `transactionHash`,
  `targetConfirmation`,
  `contractAddress`,
  `event`,
  `content`,
  `emitted`,
  `createdAt`,
  `updatedAt`
FROM
  `web3events_event` AS `web3events_event`
WHERE
  `web3events_event`.`contractAddress` = '0xddb64fe46a91d46ee29420539fc25fd07c5fea3e'
  AND `web3events_event`.`emitted` = 0
  AND `web3events_event`.`blockNumber` <= 1998;
[DEBUG] 04/01 03:33:54 (db): Executing SQL (default):
DELETE FROM
  `web3events_event`
WHERE
  `emitted` = 1
  AND `contractAddress` = '0xddb64fe46a91d46ee29420539fc25fd07c5fea3e'
  AND `blockNumber` <= 1988 - `targetConfirmation`
[INFO] 04/01 03:33:54 (web3events:storage.staking:): Fetching and processing events from block 1998 to 1998
[DEBUG] 04/01 03:33:54 (web3events:storage.staking:): Batch size is 120
[VERBOSE] 04/01 03:33:54 (web3events:storage.staking:confirmator): Running Confirmation routine
[DEBUG] 04/01 03:33:54 (db): Executing SQL (default):
SELECT
  `id`,
  `blockNumber`,
  `transactionHash`,
  `targetConfirmation`,
  `contractAddress`,
  `event`,
  `content`,
  `emitted`,
  `createdAt`,
  `updatedAt`
FROM
  `web3events_event` AS `web3events_event`
WHERE
  `web3events_event`.`contractAddress` = '0x4339316e04cffb5961d1c41fef8e44bfa2a7fbd1'
  AND `web3events_event`.`emitted` = 0
  AND `web3events_event`.`blockNumber` <= 1998;
[INFO] 04/01 03:33:54 (web3events:storage.storageManager:confirmator): Confirmed 0 events.
[DEBUG] 04/01 03:33:54 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (NULL)
[VERBOSE] 04/01 03:33:54 (web3events:storage.storageManager:): Will process 1 batches
[DEBUG] 04/01 03:33:54 (db): Executing SQL (default):
DELETE FROM
  `web3events_event`
WHERE
  `emitted` = 1
  AND `contractAddress` = '0x4339316e04cffb5961d1c41fef8e44bfa2a7fbd1'
  AND `blockNumber` <= 1988 - `targetConfirmation`
[INFO] 04/01 03:33:54 (web3events:storage.staking:confirmator): Confirmed 0 events.
[DEBUG] 04/01 03:33:54 (db): Executing SQL (default):
UPDATE
  `web3events_event`
SET
  `emitted` = $ 1,
  `updatedAt` = $ 2
WHERE
  `id` IN (NULL)
[VERBOSE] 04/01 03:33:54 (web3events:storage.staking:): Will process 1 batches
[DEBUG] 04/01 03:33:54 (web3events:storage.storageManager:): Received events for the batch: 
[INFO] 04/01 03:33:54 (web3events:storage.storageManager:): No events to be processed.
[INFO] 04/01 03:33:54 (web3events:storage.storageManager:): Finished fetching events in 0s
[DEBUG] 04/01 03:33:54 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.storageManager.lastFetchedBlockHash',
    '0x17601d78d1afd8f11c0fe638e9cc9180d55dc5d4f5598762a57004a2942b16b1'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:54 (web3events:storage.staking:): Received events for the batch: 
[INFO] 04/01 03:33:54 (web3events:storage.staking:): No events to be processed.
[INFO] 04/01 03:33:54 (web3events:storage.staking:): Finished fetching events in 0s
[DEBUG] 04/01 03:33:54 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.storageManager.lastFetchedBlockNumber',
    '1998'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:54 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.staking.lastFetchedBlockHash',
    '0x17601d78d1afd8f11c0fe638e9cc9180d55dc5d4f5598762a57004a2942b16b1'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
[DEBUG] 04/01 03:33:54 (db): Executing SQL (default):
INSERT INTO
  `data-store` (`key`, `value`)
VALUES
  (
    'web3events.storage.staking.lastFetchedBlockNumber',
    '1998'
  ) ON CONFLICT (`key`) DO
UPDATE
SET
  `key` = EXCLUDED.`key`,
  `value` = EXCLUDED.`value`;
```

--- a/package.json
+++ b/package.json
@@ -67,7 +67,7 @@
     "@rsksmart/rns-auction-registrar": "1.0.2",
     "@rsksmart/rns-reverse": "^1.0.2",
     "@rsksmart/rns-rskregistrar": "^1.2.1",
-    "@rsksmart/web3-events": "^0.1.0-dev.4",
+    "@rsksmart/web3-events": "^0.1.0-dev.6",
     "abi-decoder": "^2.3.0",
     "async-sema": "^3.1.0",
     "bignumber.js": "^9.0.0",

--- a/src/services/rns/index.ts
+++ b/src/services/rns/index.ts
@@ -153,7 +153,7 @@ const rns: CachedService = {
     const rnsReverseEventsEmitter = getEventsEmitterForService(REVERSE, web3events, rnsReverseContractAbi.abi as AbiItem[])
     const rnsPlacementsEventsEmitter = getEventsEmitterForService(PLACEMENT, web3events, simplePlacementsContractAbi.abi as AbiItem[])
 
-    const groupOptions = config.get<GroupEmitterOptions>('rns.groupEmitter')
+    const groupOptions = Object.assign({}, config.get<GroupEmitterOptions>('rns.groupEmitter'))
     const rnsGroupEmitter = web3events.groupEventsEmitters([rnsOwnerEventsEmitter, rnsReverseEventsEmitter, rnsPlacementsEventsEmitter], groupOptions)
 
     const rnsEventParser = eventTransformerFactory(rnsContractAbi.abi as AbiItem[], rnsReverseContractAbi.abi as AbiItem[], simplePlacementsContractAbi.abi as AbiItem[])

